### ISSUE TITLE
post:interactive-blogs,par:bXlpbXBsZW1lbnRh

### ISSUE THREAD

**Issue by solson** [arxanas/blog#23]
One potential advantage of using GitHub issues rather than GitHub discussions is that, if someone opens a suggested *change* or *improvement* to the article, that comment thread can be closed as resolved and hidden from the article after it is updated.

In fact, the documentation system I use at work allows commenting on individual paragraphs just like this, represents the comments as an issue in the bug tracker (assigned to the documentation owners), and indeed hides the threads once resolved.

Thinking about documentation vs. personal blogging, I like to have fairly strict control over the content published on my own website. Even for comments which aren't strictly suggestions, I would lean towards incorporating the most meaningful comments into the work itself (with attribution) whether by an extra sentence, a footnote, an `<aside>`, etc, and again closing the comment thread as resolved.

**Comment by arxanas** [comment]
> that comment thread can be closed as resolved and hidden from the article after it is updated.

You could also hide discussions by attaching labels.

> In fact, the documentation system I use at work allows commenting on individual paragraphs just like this

Which documentation system is this, if it's public?

**Comment by solson** [comment]
> You could also hide discussions by attaching labels.

True. My intention was just to emphasize a particular way one could choose to treat inline comments (as unresolved issues with the article, like code review comments in systems that track comment thread resolution).

> Which documentation system is this, if it's public?

Unfortunately it's internal-only.
